ivantiitsm_connector.py

```diff
@@ -143,8 +144,7 @@ def _suds_to_dict(self, sud_obj):
         elif isinstance(sud_obj, datetime):
             return sud_obj.strftime(consts.HEAT_TIME_FORMAT)
 
-        # Checking for NaN
-        elif sud_obj != sud_obj:
+        elif math.isnan(sud_obj):
```

Comment:
This is done to fix SemGrep warning. 
This alternative fix is suggested as part of this post https://stackoverflow.com/questions/944700/how-can-i-check-for-nan-values